functions.php

```diff
@@ -219,3 +219,8 @@ function nightingale_wp_scripts() {
  * Gravity Forms styling.
  */
 require get_template_directory() . '/inc/gravity-forms.php';
+
+/**
+ * Advanced Custom Fields.
+ */
+require get_template_directory() . '/inc/advanced-custom-fields.php';
```

Comment:
What does this inclusion enable?

Reply:
It stops ACF disabling standard custom fields (the issue we discussed last Friday).

Reply:
Can we add a comment to this extent?